pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -82,16 +83,21 @@ func (c *Cluster) Start() error {
 	logger.Infof("start running mgr")
 
 	for i := 0; i < c.Replicas; i++ {
```

Comment:
What about setting `Replicas` in the `New()` to `2` again?

Reply:
i'd prefer to keep that in a separate issue so we're just renaming here. Want to open the issue for that?

Reply:
Sure, see #1796.